### DISCUSSION TITLE
Postal 2.0 logs

### DISCUSSION THREAD

**Discussion by mjml1982** [postalserver/postal#1820] [Help with using Postal]
Hello, 
Where we can find the logs for postal 2.0?

Thanks

**Comment by willpower232** [comment]
You should be able to use `postal logs...` with the name of the container so `postal logs web` or `postal logs worker`.

Some of the UI logs are inside the container so you would need to `postal bash web` and find the logs under `/opt/postal`

**Comment by eZwzAEkx** [comment]
Hi Will,

is there a way to get logs from the last hour or for example only 100 last lines?

**Reply by willpower232** [reply]
The logs only exist as long as the container is running so if you have restarted it recently then they may no longer be available.

As Postal uses docker compose, you could get the last 100 lines of smtp logs with something like `docker-compose logs smtp --tail 100`
https://docs.docker.com/compose/reference/logs/

Unfortunately I don't think the `postal logs...` command would handle the extra arguments currently.

**Reply by AlexITC** [reply]
> Unfortunately I don't think the postal logs... command would handle the extra arguments currently.

Actually, it is supported now, I just tested it.

**Reply by willpower232** [reply]
ah yes I see now, it does pass through all arguments to docker-compose :ok_hand:
